src/openrct2/actions/TrackDesignAction.cpp

```diff
@@ -221,9 +221,13 @@ GameActionResult::Ptr TrackDesignAction::Execute() const
         ride->vehicle_colours[i].Ternary = _td.vehicle_additional_colour[i];
     }
 
-    auto gameAction = RideSetNameAction(ride->id, _td.name);
-    gameAction.SetFlags(GetFlags());
-    GameActions::ExecuteNested(&gameAction);
+    for (int32_t count = 0; count == 0 || r->Error != GA_ERROR::OK; ++count)
```

Comment:
I wonder if it's better to start with `1`? With `0`, the first ends up e.g. `Crazy Mice`, while the second one gets `Crazy Mice 1`, third `Crazy Mice 2`, etc. It would be more intuitive if the second one were to be assigned `2` as well.

Reply:
Done!